scripts/collector/postgres/db-machine-specs.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### Setup directories needed for execution
+#############################################################################
+
+# Validate the number of arguments
+if [ "$#" -ne 5 ]; then
+    echo "Usage: $0 <machine_name> <pkey> <dma_source_id> <dma_manual_id> <outputPath>"
+    exit 1
+fi
+
+# Inputs
+machine_name=$1
+pkey=$2
+dmaSourceId=$3
+dmaManualId=$4
+outputPath=$5
+
+# Function to echo log messages
+function writeLog() {
+    echo "$(date +"%Y-%m-%d %H:%M:%S") - $1"
+}
+
+# Output headers
+headers="pkey|dma_source_id|dma_manual_id|MachineName|PhysicalCpuCount|LogicalCpuCount|TotalOSMemoryMB"
```

Comment:
```suggestion
headers="pkey|dma_source_id|dma_manual_id|machine_name|physical_cpu_count|logical_cpu_count|total_os_memory_mb"
```

Can we use snake case for the column headers to be consistent with the other files?